content/posts/jenkins-snyk.md

```diff
@@ -0,0 +1,357 @@
+---
+author:
+  name: "Casey Vega"
+title: "Safety First with Jenkins and Snyk"
+date: 2019-07-31T07:00:00-00:00
+showDate: true
+draft: true
+tags: ["cloudbees","jenkins","snyk","security","vulnerability","infosec", "devsecops"]
+---
+
+## Implementing security scanning as a preventative measure in your CI pipeline.
+
+![Jenkins + Snyk = Love Security](/img/jenkins-snyk/jenkins-snyk.png)
+
+### Overview
+Today’s developers are being empowered to expeditiously innovate, creating new software capabilities and building continuous customer value. At CloudBees, we commonly tell our customers:
+
+> "Every business is a software business and is under pressure to innovate constantly. This increased velocity introduces new business risks. CloudBees is building the world's first end-to-end automated software delivery system (SDM), enabling companies to balance governance and developer freedom."  
+
+Many of the products getting built today contain or rely on sensitive consumer data and tend to have wide-spread privacy implications should the consumer's data get leaked. Unfortunately, in today's world, we're seeing more and more companies fall victim to security vulnerabilities, and the exposure of sensitive data is also happening with a semi-regular cadence. 
+
+The Equifax Breach, which resulted in an up to $700 million settlement last week, could have likely been avoidable had a mechanism existed that made mitigating security issues reliable and straightforward for developers. In this case, a lapse in process and legacy code get attributed with the blame.  
+
+Recent supply chain and typosquatting attacks against popular open-source libraries and repositories are also becoming a more significant attack vector. A simple typo or desire to stay current with the latest stable release can have a significant impact on business security and customer privacy.
```

Comment:
I think it would be useful to link to a definition/example of those supply chain/typosquatting attacks

Reply:
Poor NPM maintainers. Good call